### PR TITLE
Arducam models added

### PR DESCRIPTION
**Pull request by cafemoloko**
Here's the preview:

image
image

cc: @Luxonis-Brandon, @Luxonis-David

### CODE REVIEW

**Comment by Luxonis-David** [comment]
@cafemoloko we should remove the RPi camera adapter from page, because it is compatible with 1098FFC only which is EOL.  
We are now connecting Arducam cameras with only 
DM1090FFC-KIT
adapter cable to the 1090FFC, and we should also change the pictures accordingly.

**Comment by Luxonis-Brandon** [comment]
(And good idea to add the ArduCam cameras - thanks, all.)

**Comment by cafemoloko** [comment]
Thanks, @Luxonis-David, for the comment. I'll change it accordingly.

**Comment by cafemoloko** [comment]
@Luxonis-David - changed to the following:
cc: @Luxonis-Brandon 
image

image

File: docs/source/index.rst [hunk 1]
```diff
@@ -146,6 +146,16 @@ Cameras
      - :ref:`Learn more <bg0249>` 
      - :ref:`Learn more <bw0253>`  
 
+.. list-table::
+   :widths: 30 
+   :header-rows: 1
+
+   * - Ardcuam Cameras
```
Comment (Erol444) [inline docs/source/index.rst:153]: Arducam

**Review by Erol444 — CHANGES_REQUESTED** [review]
Could we separate cameras that are meant for FCC and cameras that are for other models? Also, it would be great to write a short tutorial on how to change the camera on eg. OAK-D (heat up the glue so you can take off the old cam? should we use super glue to glue the new cam on? Afterwards we should calibrate it - link to that site)

File: docs/source/index.rst (hunk 1 above)
Comment (cafemoloko) [reply docs/source/index.rst]: changed to ArduCam

**Comment by cafemoloko** [comment]
> Could we separate cameras that are meant for FCC and cameras that are for other models? Also, it would be great to write a short tutorial on how to change the camera on eg. OAK-D (heat up the glue so you can take off the old cam? should we use super glue to glue the new cam on? Afterwards we should calibrate it - link to that site)

Yes, good points, @Erol444 . However, are you open to merge this PR as it is right now? We'll work on the update.

**Review by Erol444 — APPROVED** [review]

**Pull request merged**